pkgs/applications/file-managers/mc/default.nix

```
@@ -100,7 +100,7 @@ stdenv.mkDerivation rec {
  '';
 
   meta = with lib; {
-    description = "File Manager and User Shell for the GNU Project";
+    description = "File Manager and User Shell for the GNU Project - midnight-commander";
```

Definitely good idea, but I'd prefer the description to be more human readable:
```suggestion
    description = "File Manager and User Shell for the GNU Project, known as Midnight Commander";
```

---

Could you include this @06kellyjac ? It's ready to merge otherwise.

---

Up 😇 that would be a welcome improvement